README.md

```diff
@@ -30,6 +30,57 @@ make demo
 The build script will then download data file for Doom automatically. SDL2 based window
 should appear when Doom is loaded and executed.
 
+## riscv-arch-test
+
+The RISC-V Architectural Tests, [riscv-arch-test](https://github.com/riscv-non-isa/riscv-arch-test), is the basic set of tests that can ensure the riscv model's behaviour that matches RISC-V specifications while implementing specific software.(not a **substitute for rigorous design verification**)
+
+There have **reference signatures** that are generated by the formal RISCV model [RISC-V SAIL](https://github.com/riscv/sail-riscv) and the test files *.elf. The .elf files contain several testing instructions, data, and **signatures**. The **test signatures** are the specific data location that need to be written by testing model(rv32emu-next) during the test. Once the test is executed, the **test signatures** will be written and compared to the **reference signature**. The test will pass if both signatures exactly match.
```

Comment:
Don't say "*.elf"

Instead, introduce "suffix" to rewrite. See https://en.wikipedia.org/wiki/Filename_extension